### PR TITLE
change to aiohttp boilerplate

### LINKED ISSUE #3: Update boilerplate for aiohttp services

### PR DESCRIPTION
**Pull request by steersbob**
Resolves #3

### CODE REVIEW

File: YOUR_PACKAGE/__main__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+from aiohttp import web
+
+from brewblox_service import events, service
+
+routes = web.RouteTableDef()
+LOGGER = logging.getLogger(__name__)
+
+
+@routes.get('/hello')
```
Comment (elcojacobs) [inline YOUR_PACKAGE/__main__.py]: maybe more descriptive function names would be better?

hello -> '/endpoint_name'
greet -> 'endpoint_implementation'

File: YOUR_PACKAGE/__main__.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+async def greet(request: Type[web.Request]):
+    return web.Response(body='Hello world!')
+
+
+async def on_message(queue, key, message):
```
Comment (elcojacobs) [inline YOUR_PACKAGE/__main__.py]: add docstrings to describe what on_message does (mentioning rabbitmq etc) and document arguments.
also applies to function above.

File: YOUR_PACKAGE/__main__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    LOGGER.info(f'Message from {queue}: {key} = {message} ({type(message)})')
+
+
+def main():
+    app = service.create()
+
+    # Add event handling
+    events.setup(app)
+    listener = events.get_listener(app)
+    listener.subscribe('brewblox', '#', on_message=on_message)
```
Comment (elcojacobs) [inline YOUR_PACKAGE/__main__.py]: add comment about wildcards and link to documentation for it. There is no mention of rabbitmq or when the plugin should use events and when it should use rest.

File: YOUR_PACKAGE/__main__.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+    # Register own routes
+    app.router.add_routes(routes)
```
Comment (elcojacobs) [inline YOUR_PACKAGE/__main__.py]: document how endpoints are registered in Janus (service name prefix / direct passing). Only host lookup or nesting.

File: YOUR_PACKAGE/__main__.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+    # Add all default endpoints, and announce service to gateway
```
Comment (elcojacobs) [inline YOUR_PACKAGE/__main__.py]: What are default the endpoints?

File: test/conftest.py [hunk 6]
```diff
@@ -22,32 +22,35 @@ def log_enabled():
 def app_config() -> dict:
```
Comment (elcojacobs) [inline test/conftest.py:22]: rename to app_config_default?

**Review by elcojacobs — CHANGES_REQUESTED** [review]

File: YOUR_PACKAGE/__main__.py (hunk 1 above)
Comment (steersbob) [reply YOUR_PACKAGE/__main__.py]: Fixed

File: test/conftest.py (hunk 6 above)
Comment (steersbob) [reply test/conftest.py:22]: You can override it - and then you'd have to create an override called `app_config_default`.

File: YOUR_PACKAGE/__main__.py (hunk 2 above)
Comment (steersbob) [reply YOUR_PACKAGE/__main__.py]: Fixed

File: YOUR_PACKAGE/__main__.py (hunk 3 above)
Comment (steersbob) [reply YOUR_PACKAGE/__main__.py]: Fixed

File: YOUR_PACKAGE/__main__.py (hunk 4 above)
Comment (steersbob) [reply YOUR_PACKAGE/__main__.py]: fixed

File: YOUR_PACKAGE/__main__.py (hunk 5 above)
Comment (steersbob) [reply YOUR_PACKAGE/__main__.py]: Added. Intentionally left out `/_debug/publish`

File: test/conftest.py (hunk 6 above)
Comment (elcojacobs) [reply test/conftest.py:22]: ok, got it. Then it is better to leave it like this.

**Review by elcojacobs — APPROVED** [review]
Great, no further comments.

**Pull request merged**